src/program.cpp

```diff
@@ -925,8 +951,14 @@ void program::perf_report(
     {
         overhead_vec.push_back(time<milliseconds>([&] { dry_run(params); }));
     }
-
     double total_time             = common_average(total_vec);
+    double min_time               = total_vec.front();
+    double max_time               = total_vec.back();
+    double mean_time              = mean(total_vec);
+    double median_time            = median(total_vec);
```

Comment:
Isnt the median just `percentile(total_vec, 0.5)`?

Reply:
That would be slightly off for the even number of runs case I think